### DISCUSSION TITLE
Range Slider ?

### DISCUSSION THREAD

**Discussion by sangelxyz** [thaw-ui/thaw#305] [General]
Is there anyway to implement a range slider in Thaw, what i mean is similar to the below where you can set the min and max values.

![Screenshot from 2024-11-17 07-18-02](https://github.com/user-attachments/assets/79ec9954-876e-4767-aff3-5330ad102eb9)

**Comment by lizidev** [comment]
Thaw-v0.4.2 has added RangeSlider component.

**Reply by sangelxyz** [reply]
Wow nice.
